.circleci/config.yml

```diff
@@ -148,4 +202,14 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - igor/nightly-build
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
What timezone is it? UTC? If so maybe we should use even less busy hours.

Reply:
UTC.
02:00 am would be better?

Reply:
UTC 2.00 is still quite early for those working in CEST. What about UTC 5.00? Seems early enough before busy hours in Cambridge at winter.